controllers/api.controller.js

```diff
@@ -14,4 +14,17 @@ function getTopics(req, res, next) {
     .catch(next)
 }
 
-module.exports = { getApi, getTopics }
+function getArticleByID(req, res, next) {
+  const { article_id } = req.params
+  fetchArticleById(article_id)
+    .then((article) => {
+      //console.log(article)
+      if (!article) {
```

Comment:
Remember that .then() is for handling a promises resolving - i.e. we got the data everything is successful - and .catch is to handle errors - theres a problem and the promise has rejected - therefore we should make sure the model resolves or rejects so the controller can send the appropriate response in the appropriate place!